cmd/finch/nerdctl.go

```diff
@@ -501,3 +676,27 @@ var nerdctlCmds = map[string]string{
 	"volume":    "Manage volumes",
 	"wait":      "Block until one or more containers stop, then print their exit codes",
 }
+
+var cmdFlagSetMap = map[string]map[string]sets.Set[string]{
```

Comment:
nit: all 3 of these maps are identical, is there a reason to have separate copies for each command name?

Reply:
Technically, each of the commands have distinct sets of flags.
However, with this first commit, I've maintained parity with the previous functionality.

This actually sets the stage for improved, nuanced handling of the various flag options for each of the present commands and any additional commands we may need to add.

Additionally, the implementation precedence within this file has established a certain amount of custom implementation for cli parsing rather than extending the dependency upon COBRA to functionally parse the cli elements.
We may or may not continue expanding our custom implementation, but that requires another level of design/plan effort.

Reply:
Makes sense, figured it was most likely to provide extensibility to each of these commands having their own unique set of flags in the future.